apps/node_monitoring/src/monitors/resource.rs

```diff
@@ -441,13 +441,13 @@ impl ResourceMonitor {
                     };
 
                     ResourceUtilization {
-                        timestamp: chrono::Local::now().timestamp(),
+                        timestamp: OffsetDateTime::now_utc().unix_timestamp(),
```

Comment:
This time is what is reported in the FE explorer. Shouldn't we keep the local time? On the other hand, maybe we should unify everything under UTC, just as you are saying here: https://github.com/tezedge/tezedge/pull/967/files#r818413396

The block timestamps are in UTC, so maybe it is the wisest to use UTC everywhere

Reply:
+1 for UTC everywhere

Reply:
The timestamp should be the same, whether it is obtained from local now or from UTC now, to my understanding.